RadixWallet/Features/ImportMnemonic/VerifyMnemonic/VerifyMnemonic+View.swift

```diff
@@ -0,0 +1,154 @@
+import SwiftUI
+
+extension VerifyMnemonic.State {
+	public var viewState: VerifyMnemonic.ViewState {
+		let enumeratedWords = mnemonic.words.identifiablyEnumerated()
+		let wordViewStates = enumeratedWords.map { word in
+			let isConfirmationWord = wordsToConfirm.contains(word)
+			return VerifyMnemonic.ViewState.WordViewState(
+				word: word,
+				placeholder: isConfirmationWord ? "" : "•••••",
+				isDisabled: !isConfirmationWord,
+				displayText: {
+					if let confirmationWord = enteredWords[id: word.id]?.element {
+						confirmationWord
+					} else {
+						""
+					}
+				}()
+			)
+		}
+
+		return .init(
+			words: wordViewStates.asIdentifiable(),
+			focusedField: focusedField,
+			verificationFailed: invalidMnemonic
+		)
+	}
+}
+
+extension VerifyMnemonic {
+	public struct ViewState: Sendable, Equatable {
+		struct WordViewState: Sendable, Equatable, Identifiable {
+			var id: OffsetIdentified<Mnemonic.Word> {
+				word
+			}
+
+			let word: OffsetIdentified<Mnemonic.Word>
+			let placeholder: String
+			let isDisabled: Bool
+			let displayText: String
+		}
+
+		let words: IdentifiedArrayOf<WordViewState>
+		let focusedField: Int?
+		let verificationFailed: Bool
+	}
+
+	@MainActor
+	public struct View: SwiftUI.View {
+		@FocusState private var focusedField: Int?
+		private let store: StoreOf<VerifyMnemonic>
+
+		public init(store: StoreOf<VerifyMnemonic>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+				ScrollView {
+					VStack(spacing: .medium1) {
+						// FIXME: Strings
+						Text(L10n.ConfirmMnemonicBackedUp.subtitle)
+							.foregroundStyle(.app.gray1)
+							.textStyle(.body1Regular)
+							.multilineTextAlignment(.center)
+
+						wordsGrid(viewStore: viewStore)
+
+						if viewStore.verificationFailed {
+							// FIXME: Strings
+							WarningErrorView(text: "Incorrect seed phrase", type: .error)
+						}
+
+						Button(L10n.Common.confirm) {
+							viewStore.send(.confirmSeedPhraseButtonTapped)
+						}
+						.buttonStyle(.primaryRectangular)
+					}
+					.padding(.horizontal, .medium2)
+					.padding(.bottom, .large3)
+				}
+				.scrollIndicators(.hidden)
+			}
+			// FIXME: Strings
+			.navigationTitle(L10n.ConfirmMnemonicBackedUp.title)
+		}
+
+		@ViewBuilder
+		private func wordsGrid(viewStore: ViewStoreOf<VerifyMnemonic>) -> some SwiftUI.View {
+			SwiftUI.Grid(horizontalSpacing: .small2, verticalSpacing: .medium1) {
```

Comment:
Use of Grid instead of LazyVGrid so that the content properly scrolls to last word.
Otherwise, if the random 3 words are from first 10 words, the last word would have been loaded due to Lazy mechanism, thus the focus on the last word will not happen until the user explicitly scrolls so the view is loaded.